src/Agent.Listener/Configuration/NativeWindowsServiceHelper.cs

```diff
@@ -45,8 +46,7 @@ public interface INativeWindowsServiceHelper : IAgentService
 
     public class NativeWindowsServiceHelper : AgentService, INativeWindowsServiceHelper
     {
-        // TODO: Change it to VSTS_AgentService_G?
-        private const string AgentServiceLocalGroupPrefix = "TFS_BuildService_G";
+        private const string AgentServiceLocalGroupPrefix = "VSTS_AgentService_G";
```

Comment:
Does adding an extra character hurt anything? IIRC, UI and API restrictions do not match.